infra/modules/multichain/main.tf

```diff
@@ -101,6 +101,14 @@ resource "google_cloud_run_v2_service" "node" {
         name  = "AWS_DEFAULT_REGION"
         value = var.indexer_options.s3_region
       }
+      env {
+        name  = "MPC_RECOVERY_GCP_PROJECT_ID"
+        value = var.project
+      }
+      env {
+        name  = "MPC_RECOVERY_SK_SHARE_SECRET_ID"
```

Comment:
should we rename these to be multichain instead of mpc-recovery?

Reply:
Mmm, not sure, maybe? I don't have a strong opinion so I just kept it as is for now

Reply:
Made a ticket for this one too: https://github.com/near/mpc-recovery/issues/407